src/Service/Resolvers/PostgresQueryBuilder.cs

```diff
@@ -110,10 +110,10 @@ public string Build(SqlUpsertQueryStructure structure)
         {
             // https://stackoverflow.com/questions/42668720/check-if-postgres-query-inserted-or-updated-via-upsert
             // relying on xmax to detect insert vs update breaks for views
-            string predicates = JoinPredicateStrings(Build(structure.Predicates), structure.DbPolicyPredicates);
+            string updatePredicates = JoinPredicateStrings(Build(structure.Predicates), structure.DbPolicyPredicatesForOperations[Config.Operation.Update]);
```

Comment:
>string updatePredicates  [](http://example.com/codeflow?start=12&length=24)

nit: i would also add a comment here reiterating the limitation you noted in "known-issues.md" that PG only supports update db policy for PUT/PATCH.

Reply:
Actually, I am removing the db policy support for PG even for update, since it does not follow the behavior described. Refer to the PR description for that.

Reply:
That would be a regression/breaking change, yes? Let's avoid that if possible

Reply:
It is a breaking change, but the behavior is not what we define for database policy in PUT/PATCH. I am not sure if we should still keep it. Thoughts @Aniruddh25 ?